stack/base/before-notebook.d/40_prepare-aiida.sh

```diff
@@ -1,15 +1,12 @@
 #!/bin/bash
 
 # This script is executed whenever the docker container is (re)started.
-
-# Debugging.
 set -x
 
-# Environment.
 export SHELL=/bin/bash
 
 # Check if user requested to set up AiiDA profile (and if it exists already)
-if [[ ${SETUP_DEFAULT_AIIDA_PROFILE} == true ]] && ! verdi profile show ${AIIDA_PROFILE_NAME} &> /dev/null; then
```

Comment:
If the profile already exists, it will get printed here (and we don't need to print it again below)